transactionoutbox-spring/pom.xml

```diff
@@ -59,18 +62,22 @@
     <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
+      <scope>test</scope>
```

Comment:
Were these dependencies deliberately not "test"?

Reply:
They're set to `test` in the parent `dependencyManagement`.

https://github.com/gruelbox/transaction-outbox/blob/master/pom.xml#L129